ios-shoppe-demo/ios-shoppe-demo/ios-shoppe-demo/ShoppeItem.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ShoppeItem.swift
+//  ios-shoppe-demo
+//
+//  Created on 3/4/20.
+//  Copyright © 2020 FullStory All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class ShoppeItem {
+    var title: String
+    var description: String
+    var price: String
```

Comment:
The "product" type has been updated: https://github.com/patrick-fs/reactshoppe/blob/develop/src/types/product.ts

![image](https://user-images.githubusercontent.com/45576380/75912248-e27b9080-5e1e-11ea-96e2-fe57dd1b28c1.png)

Reply:
What is the Unit used for?